homework/shelkovich_ilya/palindrome/src/test/java/com/tutrit/java/quickstart/service/PalindromeTest.java

```diff
@@ -0,0 +1,31 @@
+package com.tutrit.java.quickstart.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PalindromeTest {
+
+    Palindrome palindrome;
+
+    @Before
+    public void setUp(){
+        this.palindrome = new Palindrome();
+    }
+
+    @Test
+    public void shouldReturnTrue(){
+        String text = "teNneT";
+        palindrome.isPalindrome(text);
+        Assert.assertTrue(true);
```

Comment:
Do you mean 
```
Assert.assertTrue(palindrome.isPalindrome(text));
```
?

Reply:
yeah, but didn't get it how to write this)